packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -3384,7 +3384,190 @@ curl -v -X PUT \
     }
 }
 ```
+#### Update application level profile attributes
 
+Updates the Application profile attributes
+
+##### Request Parameters
+
+| Parameter     | Description                                                             | Param Type | DataType                                      | Required | Default |
+| ------------- | ----------------------------------------------------------------------- | ---------- | --------------------------------------------- | -------- | ------- |
+| app           | app with new profile attributes                                         | Body       | [Application](#application-object)             | TRUE    |         |
+| applicationId | `id` of an [app](#application-object)                                    | URL        | String                                        | TRUE     |         |
+
+##### Response parameters
+
+[Application](#application-object) with updated `profile attributes`
+
+##### Request example
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "name": "oidc_client",
+  "label": "oauth2 client app 1",
+  "signOnMode": "OPENID_CONNECT",
+  "credentials": {
+    "oauthClient": {
+      "client_id": 0oa1ivcatpQDkMraA1d7,
+      "autoKeyRotation": true,
+      "token_endpoint_auth_method": "client_secret_post"
+    }
+  },
+  "profile": {
+          "label": "oauth2 client app 1"
+      },
+  "settings": {
+    "oauthClient": {
+      "client_uri": "http://localhost:8080",
+      "logo_uri": "http://developer.okta.com/assets/images/logo-new.png",
+      "redirect_uris": [
+        "https://example.com/oauth2/callback",
+        "myapp://callback"
+      ],
+      "post_logout_redirect_uris": [],
+      "response_types": [
+        "token",
+        "id_token",
+        "code"
+      ],
+      "grant_types": [
+        "implicit",
+        "authorization_code"
+      ],
+      "consent_method": "TRUSTED",
+      "issuer_mode": "ORG_URL",
+      "application_type": "native",
+      "tos_uri": "",
+      "policy_uri": ""
+    }
+  }
+}' "https://${yourOktaDomain}/api/v1/apps/0oabkvBLDEKCNXBGYUAS"
+```
+
+##### Response example
+
+```json
+{
+  "id": "0oa1ivcatpQDkMraA1d7",
+  "name": "oidc_client",
+  "label": "oauth2 client app 1",
+  "status": "ACTIVE",
+  "lastUpdated": "2021-10-04T22:48:42.000Z",
+  "created": "2021-10-04T22:42:38.000Z",
+  "accessibility": {
+    "selfService": false,
+    "errorRedirectUrl": null,
+    "loginRedirectUrl": null
+  },
+  "visibility": {
+    "autoLaunch": false,
+    "autoSubmitToolbar": false,
+    "hide": {
+      "iOS": true,
+      "web": true
+    },
+    "appLinks": {
+      "oidc_client_link": true
+    }
+  },
+  "features": [],
+  "signOnMode": "OPENID_CONNECT",
+  "credentials": {
+    "userNameTemplate": {
+      "template": "${source.login}",
```

Comment:
The $ symbol means that a dev must input a variable - is that what you mean to convey here?

Reply:
Yes, thats correct. In this instance, it is just showing what the userNameTemplate should be. This is from the sample response of new API.
you can also find this application object sample response here https://developer.okta.com/docs/reference/api/apps/#example